lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_consumer.erl

```diff
@@ -145,6 +145,16 @@ on_start(InstanceId, Config) ->
                 instance_id => InstanceId,
                 kafka_hosts => BootstrapHosts
             });
+        {error, already_present = Reason} ->
+            ?SLOG(error, #{
+                msg => "failed_to_start_kafka_consumer_client",
+                instance_id => InstanceId,
+                kafka_hosts => BootstrapHosts,
+                reason => emqx_misc:redact(Reason)
+            }),
+            throw(
+                "Kafka client failed to start properly.  Please check the connection parameters."
```

Comment:
```suggestion
                "Failed to start Kafka client. Please check the connection parameters."
```

also, should this be moved to line 165?

Reply:
You mean write this message for any error?  I was trying to be specific here, because I don't know what other errors might be returned here.